tests/helpers/intl-test-helper.jsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 import {IntlProvider, intlShape} from 'react-intl';
 import {mount, shallow} from 'enzyme';
 
-const intlProvider = new IntlProvider({locale: 'en'}, {});
+const intlProvider = new IntlProvider({locale: 'en', timeZone: 'America/Los_Angeles'}, {});
```

Comment:
0/5, but should we converge on UTC vs. the US west coast?

Reply:
I thought of keeping my local timezone first :). I am 0/5 as long as you pick the timezone and give me. 

The way i see it we are checking of they render the same timestamps for a given timezone always. I am ok with anything really.

Reply:
I'd propose converging on UTC as the canonical timezone. It's almost certainly not very important either way, but while we're in here 🤷‍♂️

Reply:
Done. True if we are going to have a default timezone better it be UTC